gifted_build/src/index.tsx

```diff
@@ -5,13 +5,15 @@ import { Route, BrowserRouter as Router } from 'react-router-dom'
 import NavBar from "./components/navbar";
 import Homepage from "./components/homepage";
 import Second_Page from "./components/second_page";
+import Special_Olympics from "./components/special_olympics";
 import Footer from "./components/footer";
 import './css/common.css'
 
 const Body = (
     <Router>
         <Route exact path ="/" component={Homepage} />
         <Route path="/2" component={Second_Page} />
+        <Route path="/3" component={Special_Olympics} />
```

Comment:
change this temporary route path from 3 to something more appropriate like "/partners" in order for SEO to work better on this site

Reply:
This is taken care of.